### PR TITLE
Update wrong comment text

### PR DESCRIPTION
**Pull request by kishanjasani**
- Update wrong comment text of caching args

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -238,8 +238,8 @@ $args = array(
 // http://codex.wordpress.org/Class_Reference/WP_Query#Caching_Parameters
 // NOTE Caching is a good thing. Setting these to false is generally not advised.
     'cache_results' => true, // (bool) Default is true - Post information cache.
-    'update_post_term_cache' => true, // (bool) Default is true - Post meta information cache.
-    'update_post_meta_cache' => true, // (bool) Default is true - Post term information cache.
+    'update_post_term_cache' => true, // (bool) Default is true - Post term information cache.
+    'update_post_meta_cache' => true, // (bool) Default is true - Post meta information cache.
```
Comment (luetkemj) [inline README.md:242]: Good catch!

File: README.md [hunk 2]
```diff
@@ -106,9 +106,9 @@ $args = array(
       'draft',  // - a post in draft status.
       'auto-draft', // - a newly created post, with no content.
       'future', // - a post to publish in the future.
-      'private', // - not visible to users who are not logged in.
+      'private', // - not visible to any users.
       'inherit', // - a revision. see get_children.
-      'trash', // - post is in trashbin (available with Version 2.9).
+      'trash', // - post is in trash/bin (available with Version 2.9).
```
Comment (luetkemj) [inline README.md]: Please remove the status-parameter changes as they do not reflect the documentation the comments are based on: https://developer.wordpress.org/reference/classes/wp_query/#status-parameters

**Review by luetkemj — CHANGES_REQUESTED** [review]
Thanks for submitting a PR! Please see comments - looking forward to merging :)

File: README.md (hunk 2 above)
Comment (kishanjasani) [reply README.md]: Ah, Sure :+1:

File: README.md [hunk 3]
```diff
@@ -106,7 +106,7 @@ $args = array(
       'draft',  // - a post in draft status.
       'auto-draft', // - a newly created post, with no content.
       'future', // - a post to publish in the future.
-      'private', // - not visible to users who are not logged in.
+      'private', // - not visible to any users.
```
Comment (luetkemj) [inline README.md]: Whoops - I wasn't clear. Please remove this one too.

**Review by luetkemj — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (kishanjasani) [reply README.md]: Logged in user also can not see this post. That's why I changed it a bit.

Comment (kishanjasani) [reply README.md]: Only the author of the post can see this post ☺

Comment (luetkemj) [reply README.md]: Private content is visible to `those with authorization permission levels to see private content`. By default this includes Editors and Administrators. https://wordpress.org/support/article/content-visibility/#private-content

How about changing line 109 to:
`'private', // - visible only to the author, administrators, and editors.`

Comment (kishanjasani) [reply README.md]: Yes, It could be better :slightly_smiling_face:

**Review by luetkemj — APPROVED** [review]
LGTM!

**Pull request merged**